# Hypothetical-Protein-Analysis-
In-silico analysis of a hypothetical protein from a resistant pathogen 
This project includes the computational analysis of a hypothetical protein sequence retrieved from NCBI.  
Tools used: NCBI, MEGA7, ProtParam, CELLO, PSORTB, CDD, SOPMA Server,Chimerax, Procheck, Verify 3D, PrankWeb, Expassy Docking.
Steps:
1. Retrieved sequence from NCBI.
2.Determined evolutionary relatedness through BLAST searches, multiple sequence alignment, and phylogenetic tree construction.
3. Analyzed physicochemical properties using ProtParam to assess the stability of the protein.
3.predicted the subcellular localization using CELLO and Psortb.
4. Identified functional domains using Conserved Domain Database (CDD).
5. Predicted the 3D structure using AlphaFold.
6. Validated the predicted 3D structure.
7.  Predicted potential active sites using PrankWeb.
8.  Performed molecular docking studies on the protein.
9. Discussed possible biological functions based on sequence features.
